includes/news/v2/mail/featured-stories.php

```diff
@@ -18,31 +18,30 @@
 			<table class="tableCollapse" width="100%" style="width: 100%; border-spacing: 0; border-collapse: collapse;"><tbody>
 				<tr>
 					<td>
-						<table width="100%" border="0" align="center" style="border-spacing: 0; border-collapse: collapse;">
-							<tbody>
-								<tr>
-									<td style="padding-left: 0; padding-right: 0;">
-										<a href="<?php echo $permalink; ?>">
+						<a href="<?php echo $permalink; ?>" style="color: #000; text-decoration: none;">
+							<table width="100%" border="0" align="center" style="border-spacing: 0; border-collapse: collapse;">
+								<tbody>
+									<tr>
+										<td style="padding-left: 0; padding-right: 0;">
 											<img class="responsiveimg" border="0" width="290" style="border:none;" src="<?php echo $image; ?>" />
-										</a>
-									</td>
-								</tr>
-								<tr>
-									<td class="montserratsemibold" style="padding-left: 0; padding-right: 0; font-family: Helvetica, Arial, sans-serif; font-size: 18px; font-weight: 500; padding-top: 10px; padding-bottom: 4px; line-height: 1.2; color: #000; text-align: left;" align="left">
-										<a href="<?php echo $permalink; ?>" style="color: #000; text-decoration: none;"><?php echo $title; ?></a>
-									</td>
-								</tr>
-								<tr>
-									<td class="montserratlight" style="padding-left: 0; padding-right: 0; font-family: Helvetica, Arial, sans-serif; font-size: 14px; font-weight: 400; line-height: 1.4; color: #000; text-align: left;" align="left">
-										<?php echo $description; ?>
-										<a href="<?php echo $permalink; ?>" style="color: #006699; text-decoration: underline;">Read More</a>
-									</td>
-								</tr>
-								<?php echo display_social_share( $permalink, $title ); ?>
-							</tbody>
-						</table>
+										</td>
+									</tr>
+									<tr>
+										<td class="montserratsemibold" style="padding-left: 0; padding-right: 0; font-family: Helvetica, Arial, sans-serif; font-size: 18px; font-weight: 500; padding-top: 10px; padding-bottom: 4px; line-height: 1.2; color: #000; text-align: left;" align="left">
+											<?php echo $title; ?>
+										</td>
+									</tr>
+									<tr>
+										<td class="montserratlight" style="padding-left: 0; padding-right: 0; font-family: Helvetica, Arial, sans-serif; font-size: 14px; font-weight: 400; line-height: 1.4; color: #000; text-align: left;" align="left">
```

Comment:
While it's unlikely to be an issue, we might want to consider explicitly stripping HTML tags (`strip_tags()`) from `$description` here, in case someone editing a Today story tries to get clever and force an `<a>` tag in this description value.  If that ever happened, the parent link would become malformed and might bork the rest of the email layout, depending on the email client.